lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -1,4 +1,27 @@
 // Helpers
+
+// Get path for named route, prefixing baseUrl if necessary
+// Uses the route definitions in /lib/gollum/views/helpers.rb.
+// For example, routePath('delete') is equivalent to 'delete_path' in the mustache templates.
+function routePath(name){
+  var routes = $.parseJSON('<%= routes_to_json %>')
+  path = routes[name]
+  if (baseUrl == undefined ) {
+    console.log('Gollum error: baseUrl undefined')
```

Comment:
When does this happen (baseUrl == undefined)? And if it happens, does it break functionality?

Reply:
It would happen if we somehow forget to set the `baseUrl` variable in the page template. I thought it would be useful to add the debug message in case that ever happens.

Reply:
👍